api_cluster_control_plane.go

```diff
@@ -95,7 +94,9 @@ func (a *SystemAdministrationPolicyClusterControlPlaneApiService) CreateOrUpdate
 	if localVarHttpResponse.StatusCode < 300 {
 		// If we succeed, return the data, otherwise pass on to decode error.
 		err = a.client.decode(&localVarReturnValue, localVarBody, localVarHttpResponse.Header.Get("Content-Type"))
-		return localVarReturnValue, localVarHttpResponse, err
+		if err == nil {
```

Comment:
What if err is not nil? It seems that it will return nil err eventually in line 118

Reply:
removed the err == nil check.